.github/workflows/run-tests.yml

```diff
@@ -40,6 +40,15 @@ jobs:
       - name: Test tools
         run: sbt "++ ${{ matrix.scala }} -v" "-no-colors" "-J-Xmx3G" test-tools
 
+        # Running all partests would take ~2h for each Scala version, run only single test of each kind
+        # to make sure that infrastructure works correctly.
+      - name: Run subset of partest tests
```

Comment:
We should also have a step to run the JUnit tests. I believe for those we can run all the tests.

Reply:
JUnit tests were added to `test-runtime` alias used in the CI. Should I move them to some other place?

Reply:
Ah no, that's fine.